src/mantarray_desktop_app/server.py

```diff
@@ -202,8 +202,14 @@ def system_status() -> Response:
 
     Can be invoked by: curl http://localhost:4567/system_status
     """
-    board_idx = 0
     shared_values_dict = _get_values_from_process_monitor()
+    if (
+        "expected_software_version" in shared_values_dict
+        and shared_values_dict["expected_software_version"] != CURRENT_SOFTWARE_VERSION
+    ):
+        return Response(status="500 Versions of Electron and Flask EXEs do not match")
```

Comment:
note for future - `500` is a catch-all code that Flask itself sometimes returns (i.e. if there was a python error while running the /system_status route code), so please update to something more specific within the 500 series